### ISSUE TITLE
problem about test result

### ISSUE THREAD

**Issue by legendary111** [cardwing/Codes-for-Lane-Detection#330]
![7e3c98362173ef9c02f165497723fb9](https://user-images.githubusercontent.com/86969009/127774351-c496e8df-c298-4fd9-aa22-70b301ff3e78.png)
Here's my test results using my own data set, and it's clear that the line is not ending, where should I change to fit my own data set? Is it the post-processing part?

**Comment by cardwing** [comment]
Do you directly apply the pretrained model to your own test set? If so, you may need to leverage some domain adaptation strategies to mitigate the performance gap.

**Comment by legendary111** [comment]
![895bea34543813e4f03cc2cbf9c67d8](https://user-images.githubusercontent.com/86969009/127940802-300b74b1-e109-4d94-bb1e-138a6473e159.png)
Thanks for your reply! I first trained my own data set with the pre-training weights, and during the training, I removed the pre-processing 240 clipping part, and adjusted the coordinate transformation. The following figure shows the latest prediction results. Loss has been kept at about 0.04 without falling, but the leftmost lane line is difficult to detect. Which part should I adjust?

**Comment by cardwing** [comment]
In your own training set, do you label the road curb? I think the rightmost lane should be easy to detect if the training set covers such conditions. The leftmost lane is also not difficult to detect. Maybe your model does not see many similar samples during training. What's the size of your dataset (train + val + test)?

**Comment by legendary111** [comment]
thanks for your reply! In my own training set , l label the road curb , but about the size of my dataset , l just label 100 images. Maybe the size of dataset is small ,l will continue to label ,train and then check the latest results.
